presto-main/src/main/java/io/prestosql/operator/TopNRowNumberOperator.java

```diff
@@ -33,7 +33,6 @@
 import static com.google.common.base.Preconditions.checkState;
```

Comment:
Please create separate PR. This commit is unrelated to rank improvements

Reply:
@sopel39, this is a prerequisite for this refactor. The rank changes are dependent on these changes to function correctly in the Builder refactor.